src/main/qss.js

```diff
@@ -649,14 +649,20 @@ gpii.app.qssWrapper.loadSettings = function (assetsManager, installedLanguages,
     var languageSetting = fluid.find_if(loadedSettings, function (setting) {
         return setting.path === settingOptions.settingPaths.language;
     });
+
     var volumeSetting = fluid.find_if(loadedSettings, function (setting) {
         return setting.path === settingOptions.settingPaths.volume;
     });
-    gpii.app.qssWrapper.populateLanguageSettingOptions(settingOptions, locale, installedLanguages, languageSetting);
 
-    // sync the language and volume value as well
-    languageSetting.value = locale;
-    volumeSetting.value = gpii.windows.nativeSettingsHandler.GetVolume().value;
+    if (fluid.isValue(languageSetting)) {
+        gpii.app.qssWrapper.populateLanguageSettingOptions(settingOptions, locale, installedLanguages, languageSetting);
+        // sync the language value as well
+        languageSetting.value = locale;
+    }
+
+    if (fluid.isValue(volumeSetting) && fluid.isValue(gpii.windows.nativeSettingsHandler)) {
```

Comment:
It seems impossible - gpii.windows.nativeSettingsHandler is loaded synchronously on process startup by means of "require" statements and I can't imagine a possible route where it is not defined. Could you try removing this and paste any stack trace you receive?

Reply:
Hahaha, you got me baby! I've meant to check if the nativeSettingsHandler exists on a different place. I will remove it ASAP ;-).